deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts

```diff
@@ -15,7 +15,10 @@ export default migration('1666906000_configurate_seed_reserves_and_rewards', {
     const trace = deploymentManager.tracer();
     const ethers = deploymentManager.hre.ethers;
 
-    const comptrollerV2 = await deploymentManager.fromDep('comptrollerV2', 'mainnet', 'usdc');
+    const comptrollerV2Proxy = await deploymentManager.fromDep('comptrollerV2', 'mainnet', 'usdc');
+    const comptrollerV2Impl = await deploymentManager.fromDep('comptrollerV2:implementation', 'mainnet', 'usdc');
```

Comment:
necessary for us to be able to call `populateTransaction` with `'_setCompSpeeds(address[],uint256[],uint256[])'` in the `proposal` function